types/cesium/index.d.ts

```diff
@@ -2949,16 +2954,16 @@ declare namespace Cesium {
         granularity: Property;
         shadows: Property;
         show: Property;
-        material: MaterialProperty;
-        positions: Property;
-        width: Property;
+        material: MaterialProperty | Color;
+        positions: Property | Cartesian3[];
```

Comment:
This breaks our application with the following error:
`error TS2339: Property 'getValue' does not exist on type 'Property | Cartesian3[].`